src/coreclr/nativeaot/System.Private.CoreLib/src/Internal/Runtime/CompilerHelpers/MemoryHelpers.cs

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime;
+using System.Runtime.CompilerServices;
+
+namespace Internal.Runtime.CompilerHelpers
+{
+    /// <summary>
+    /// These methods are used to implement memcpy and memset intrinsics with null checks.
+    /// </summary>
+    internal static class MemoryHelpers
+    {
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
FWIW, `AggressiveInlining` is effectively a no-op here, since the Jit never inlines helper calls in general, and only uses these helpers late in the phase order.

Reply:
Removed.